projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -543,6 +543,12 @@
 	background-color: var(--jetpack--contact-form--input-background);
 }
 
+/* Preserve the background color for the checkbox multiple and radio wrap when using the old checkbox multiple and radio blocks. */
+.contact-form .is-style-outlined .grunion-field-checkbox-multiple-wrap:not(.wp-block-jetpack-field-checkbox-multiple),
+.contact-form .is-style-outlined .grunion-field-radio-wrap:not(.wp-block-jetpack-field-radio) {
+    background-color: var(--jetpack--contact-form--input-background);
+}
+
```

Comment:
Just noticed that this is higher specificity than some of our other rules. However, since this only applies to non migrated markup, this sounds okay to me, since it matches the kind of specificity we had before the inner blocks migration?

Reply:
That's the gist. Only migrated mark up will contain the valid block `wp-block_*` classes.